Bio/motifs/minimal.py

```diff
@@ -0,0 +1,158 @@
+# By Ariel Aptekmann 2017
+# Adapted from Bio.motifs 2008 by Bartek Wilczynski
+# Adapted from  Bio.MEME.Parser by Jason A. Hackney.  All rights reserved.
```

Comment:
What do you mean by Bio.MEME.Parser? There isn't such a namespace in Biopython (at least currently), so what code from Jason A. Hackney are you referring to?

(Depending on the answer to this comes more questions about licensing, including can we make this new file dual licensed under the 3-clause BSD license as well).

Reply:
I don't know if this header is still applicable to this code, the full parser with other functions was inspired in parts by this http://biopython.org/DIST/docs/api/Bio.motifs.mast-pysrc.html 
Since this was already of Biopython, is there a conflict?

Reply:
Ah. That makes more sense now. Thanks.

If your code is heavily based on the older code, we're fine for using it in Biopython but only under our traditional license. However, it is a bit of a problem, as I don't think we've got Jason's permission to dual license their contributions yet: https://github.com/biopython/biopython/issues/898#issuecomment-406868039

i.e. This *may* be a problem for ``Bio/motifs/mast.py``, and by extension also for your new code.

If however all you've done is based the code structure/style on Bartek's to follow the style approach, I think you can remove this comment in the header and the licensing worry. Just use our new standard dual licensing standard header. Thanks.